indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -15,7 +15,7 @@
  * along with Indico; if not, see <http://www.gnu.org/licenses/>.
  */
 
-/* global setupListGenerator:false */
+/* global setupListGenerator:false, getParamsFromSelectors: false, initForms: False */
```

Comment:
`s/False/false` and so far we never put spaces after the `:` in a global declaration; the [example](http://eslint.org/docs/user-guide/configuring#specifying-globals) doesn't put spaces there either